app/app.go

```diff
@@ -423,11 +422,6 @@ func NewDesmosApp(
 		wasmOpts...,
 	)
 
-	// The gov proposal types can be individually enabled
-	if len(wasmEnabledProposals) != 0 {
-		govRouter.AddRoute(wasm.RouterKey, wasm.NewWasmProposalHandler(app.WasmKeeper, wasmEnabledProposals))
-	}
```

Comment:
Remove it since WasmProposalHandler was deprecated, now using gov1 to handle proposals. See:
https://github.com/CosmWasm/wasmd/blob/33df4aeaaa419f6aad07b94d64c0423bee0a2d21/x/wasm/keeper/proposal_handler.go#L19
https://github.com/CosmWasm/wasmd/issues/1246